docs/topics/Gateway_Events.md

```diff
@@ -543,6 +546,23 @@ Sent when a message is pinned or unpinned in a text channel. This is not sent wh
 | channel_id          | snowflake          | ID of the channel                                       |
 | last_pin_timestamp? | ?ISO8601 timestamp | Time at which the most recent pinned message was pinned |
 
+### Entitlements
+
+#### Entitlement Create
+
+Sent when an entitlement is created. The inner payload is a [entitlement](#DOCS_MONETIZATION_ENTITLEMENTS/entitlement-object) object.
+
+#### Entitlement Update
+
+Sent when an entitlement is renewed for the next billing period. The inner payload is an [entitlement](#DOCS_MONETIZATION_ENTITLEMENTS/entitlement-object) object. The `ends_at` field will have an updated value with the new expiration date.
```

Comment:
It's not accurate to say that the `ends_at` field will always have a new value. Entitlements could be updated for any reason.
```suggestion
Sent when an entitlement is renewed for the next billing period. The inner payload is an [entitlement](#DOCS_MONETIZATION_ENTITLEMENTS/entitlement-object) object. The `ends_at` field may have an updated value with the new expiration date.
```